sql/share/errmsg-utf8.txt

```diff
@@ -2719,7 +2721,7 @@ ER_UNKNOWN_CHARACTER_SET 42000
         swe "Okänd teckenuppsättning: '%-.64s'"
         ukr "Невідома кодова таблиця: '%-.64s'"
 ER_TOO_MANY_TABLES  
-        cze "Příliš mnoho tabulek, MariaDB jich může mít v joinu jen %d"
+        cze "Příliš mnoho tabulek, MySQL jich může mít v joinu jen %d"
```

Comment:
What is the reason for replacing MariaDB with MySQL here and in other places?

Reply:
It looks like you've used an older version of the file, from MariaDB 10.2 or earlier, so are making lots of probably unintentional reverts back to this old version of the file. Can you either target this at 10.2 only, so that it only contains changes to the Spanish translations, or alternatively do a 10.6 version, also without any non-Spanish changes.